demo/dynamic_embedding/movielens-100k-sync-estimator-with-horovod/session_hook.py

```diff
@@ -0,0 +1,42 @@
+import horovod.tensorflow as hvd
```

Comment:
It will introduce a new hvd dependency. Maybe use [NcclAllReduce](https://www.tensorflow.org/api_docs/python/tf/distribute/NcclAllReduce#broadcast) directly could avoid additional dependencies, isolated communicator problem, and cost of negotiation in horovod.